src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    public static final String SUPPLY_STR = "supply";
+    public static final String BUY_STR = "buy";
+    public static final String RESULT_STR = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] input = readStringsFromFile(fromFileName);
+
+        int supplyAmount = 0;
+        int buyAmount = 0;
+        for (String inputEntry : input) {
+            String[] inputEntrySplited = inputEntry.split(",");
+            if (inputEntrySplited[0].equals("supply")) {
+                supplyAmount += Integer.valueOf(inputEntrySplited[1]);
+            } else {
+                buyAmount += Integer.valueOf(inputEntrySplited[1]);
+            }
+        }
+
+        File toFile = createFile(toFileName);
+        writeToFile(toFile, createAReport(supplyAmount, buyAmount).getBytes());
+    }
```

Comment:
In general may look like below
```suggestion
        String[] input = readStringsFromFile(fromFileName);
        String report = createReport(input);
        writeToFile(toFileName, report);
    }
```